src/onemod/dtypes/unique_list.py

```diff
@@ -0,0 +1,25 @@
+from typing import Hashable, List, TypeVar
+
+from pydantic import AfterValidator
+from typing_extensions import Annotated
+
+T = TypeVar("T", bound=Hashable)
+
+
+def unique_list(items: List[T]) -> List[T]:
```

Comment:
Similar to `update_unique_list`, could `unique_list` be `return list(dict.from_keys(items))`?